packages/server/tests/acceptance/index.spec.ts

```diff
@@ -116,9 +116,9 @@ describe('RPC Server Acceptance Tests', function () {
 
     function runLocalHederaNetwork() {
         // set env variables for docker images until local-node is updated
-        process.env['NETWORK_NODE_IMAGE_TAG'] = '0.29.0-alpha.1';
-        process.env['HAVEGED_IMAGE_TAG'] = '0.29.0-alpha.1';
-        process.env['MIRROR_IMAGE_TAG'] = '0.62.0-rc1';
+        process.env['NETWORK_NODE_IMAGE_TAG'] = '0.30.0-alpha.0';
+        process.env['HAVEGED_IMAGE_TAG'] = '0.30.0-alpha.0';
+        process.env['MIRROR_IMAGE_TAG'] = '0.64.0-beta1';
```

Comment:
fyi, there seems to be a 0.64.0-beta2 for mirror node available now.

Reply:
Updated